com.dynamo.cr/com.dynamo.cr.bob/build.gradle

```diff
@@ -288,7 +288,8 @@ task createBobLightJar(type: Jar, dependsOn: [createExternalLibsLightJar, create
         include 'libexec/**/glslang*',
                 'libexec/**/tint*',
                 'libexec/**/spirv-opt*',
-                'libexec/**/luajit*'
+                'libexec/**/luajit*',
+                'libexec/**/aapt2'
```

Comment:
We have to use `aapt2` from `build tools` to link apk instead of one from `bundletool-all.jar`

Otherwise we will get:
> RES_TABLE_TYPE_TYPE entry offsets overlap actual entry data.
Failed to load resources table in APK ‘android.jar’.


https://github.com/google/bundletool/issues/381